src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +52,13 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        Predicate<Person> validPerson = e -> e.getAge() >= fromAge
+                && e.getAge() <= toAge
+                && e.getSex() == Person.Sex.MAN;
+
+        return peopleList.stream()
+                .filter(validPerson)
+                .collect(Collectors.toList());
```

Comment:
```suggestion
                .toList();
```
Let’s  apply it to all methods